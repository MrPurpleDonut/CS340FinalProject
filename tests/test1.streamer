num x = 3.5;
num y = x + 1;
print y;
