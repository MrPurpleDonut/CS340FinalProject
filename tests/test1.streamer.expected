Program parsed correctly!
---------------------------
--------Printing AST-------
---------------------------
Program
    Function List
    Statement List
        Declaration statement: x
            Type: num
            Assignment statement: x
                Number: 3.5
        Declaration statement: y
            Type: num
            Assignment statement: y
                Binary operator '+'
                    Identifier: x
                    Number: 1.0
        Print Statement
            Identifier: y
